examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -245,7 +245,6 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
 
     // load geoJson to get city polygons
     $scope.loadCityJsonByBound = function loadCityJsonByBound(onEachFeature, fromEventName, fromEvent){
-
```

Comment:
I compared this file with current master, and didn't see any difference between them except the last line `ng-controller`'s value. Let's avoid this kind of submission with files reverted back to original version manually which will result in a lot of format changes highlight, but actually nothing changed. Please use `git checkout -- filePath` to revert a file back to original version.